src/Extensions/ExtHostClient.re

```diff
@@ -175,7 +174,7 @@ let start =
     | ("MainThreadSCM", "$registerSourceControl", args) =>
       switch (args) {
       | [`Int(handle), `String(id), `String(label), rootUri] =>
-        let rootUri = Core.Uri.of_yojson(rootUri) |> Utility.Result.to_option;
+        let rootUri = Core.Uri.of_yojson(rootUri) |> Stdlib.Result.to_option;
```

Comment:
Unfortunately it seems we have to add `Stdlib.` to every use of `Result`, because there is an `@opam/result` polyfill package used by lots of other packages which adds a `Result` module that'll otherwise override it.